src/axom/core/Map.hpp

```diff
@@ -746,7 +860,10 @@ class Map
   float m_load_factor; /*!< currently unused value, used in STL unordered_map to determine when to resize, which we don't do internally at the moment */
   axom_map::Node<Key, T> m_end; /*!< the node with meaningless values allowing for user to verify success or failure of operations */
```

Comment:
I'd rephrase the documenting comment to something like "the sentinel node allowing the user to verify success of operations"